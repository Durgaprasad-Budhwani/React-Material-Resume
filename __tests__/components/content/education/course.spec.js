/**
 * Created by durgaprasad on 10/10/17.
 */
/* global expect,it */
import React from 'react';
import Course from '../../../../src/components/content/education/course';
import renderer from 'react-test-renderer';

it('renders course', () => {
  const tree = renderer.create(
    <Course
      year="name1"
      name="abc"
      university="'abc"
    />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});