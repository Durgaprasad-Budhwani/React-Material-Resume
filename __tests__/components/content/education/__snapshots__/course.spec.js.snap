// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders course 1`] = `
<div
  className="MuiGrid-typeContainer-4 MuiGrid-spacing-xs-16-19 MuiGrid-direction-xs-column-6"
  style={
    Object {
      "paddingLeft": 20,
    }
  }
>
  <div>
    <span
      className="Bubble-container-87"
    >
      name1
    </span>
  </div>
  <h3
    className="Course-header3-2"
  >
    abc
  </h3>
  <p
    className="Course-header4-3"
  >
    'abc
  </p>
</div>
`;
